docs/explanation/system-snaps/udisks2/install-udisks2.md

```diff
@@ -0,0 +1,33 @@
+(explanation-system-snaps-udisks2-install-udisks2)=
+# Install Udisks2
+
+
+The Udisks2 snap is currently available from the Snap Store. At can be installed on any system that supports snaps but is only recommended on [Ubuntu Core](/index) at the moment.
+
+You can install the snap with the following command:
+```bash
+snap install udisks2
+```
+All necessary plugs and slots will be automatically connected within the installation process. You can verify this with:
+
+```bash
+ snap connections udisks2
+```
+
+Once the installation has successfully finished the Udisks2 service is running in the background. You can check its current status with
+
+```bash
+systemctl status systemctl status snap.udisks2.udisksd
```

Comment:
"systemctl status" repeated :). But maybe we should suggest "snap services" instead.